src/cli/README.md

```diff
@@ -367,19 +367,61 @@ Done
 Done
 ```
 
-### br omrprefix
+### br info
 
-Get the randomly generated off-mesh-routable prefix of the Border Router.
+Print info about the border routing manager.
 
 ```bash
-> br omrprefix
+> br info
```

Comment:
Is it useful to dump all the BR information out in one command? I suspect there is more information we might want to include here. How do we decide what to include or what not to include?

Reply:
I wanted to get a snapshot of all prefixes. 
But if we change `br omrprefix` to output both local and favored, it can be used instead.
I am ok removing the `info`.